experiments/adiac_trained.py

```diff
@@ -52,18 +52,25 @@
 parser.add_argument("--input_fn", type=str, default="linear", choices=["linear", "mlp"],
                     help="input preprocessing modality")
 
+parser.add_argument("--diffusive_gamma", type=float, default=0.0, help="diffusive term")
+
 parser.add_argument("--inp_scaling", type=float, default=1.0, help="ESN input scaling")
 parser.add_argument("--use_test", action="store_true")
 parser.add_argument(
     "--trials", type=int, default=1, help="How many times to run the experiment"
 )
-parser.add_argument("--save", action="store_true", help="Save the model in the same folder as the results.")
-parser.add_argument("--load", action="store_true", help="Load the last saved model. Train only the readout.")
-parser.add_argument("--noisy", action="store_true", help="Add gaussian noise in input-to-hidden and reservoir activations.")
```

Comment:
why removing these parameters?

Reply:
This issue is my fault, caused by the version I was working on and a copy-paste mistake. I resolved it by reintroducing the missing code lines.